Tools/patchcheck/patchcheck.py

```diff
@@ -175,6 +187,15 @@ def report_modified_files(file_paths):
     'Tools/c-analyzer/cpython/_parser.py',
 })
 
+#: Python files with bad syntax
+_PYTHON_SYNTAX_PROBLEMS = frozenset({
+    'Lib/test/tokenizedata/bad_coding.py',
+    'Lib/test/tokenizedata/bad_coding2.py',
+    'Lib/test/tokenizedata/badsyntax_3131.py',
+    'Lib/test/tokenizedata/coding20731.py',
```

Comment:
To avoid hardcoding filenames, we could skip the whole `tokenizedata` directory, as we did in https://github.com/python/cpython/pull/109513.

Reply:
Fine by me!